tests/test_base.py

```diff
@@ -733,6 +735,81 @@ def test_pack_header_for_user(self, app, user_class):
             assert token_data['duder'] == 'brief'
             assert token_data['el_duderino'] == 'not brief'
 
+    def test_reset_email(self, app, user_class, db, tmpdir):
+        """
+        This test verifies email based registration functions as expected.
```

Comment:
s/registration/reset/ (copy-pasta)

Reply:
yeah, i'm extra lazy in vim at times. cleaned.

Reply:
ah, a fellow vim user. huzzah!